src/components/ModalBackupWords.js

```diff
@@ -7,12 +7,11 @@
 
 import React from 'react';
 import $ from 'jquery';
-import wallet from '../utils/wallet';
-import helpers from '../utils/helpers';
 import _ from 'lodash';
-import { WORDS_VALIDATION } from '../constants';
 import { updateWords } from '../actions/index';
 import { connect } from "react-redux";
+import hathorLib from '@hathor/wallet-lib';
```

Comment:
You could have used `import { wallet, helpers } from '@hathor/wallet-lib'`, couldn't you?

Reply:
Yes, I could..I decided to always use `hathorLib` to make clear which utils I am using because of the wallet utils.

Reply:
Ok.